scripts/doc_checker.py

```diff
@@ -524,7 +524,7 @@ def load_obj(name, old_load_obj=Docstring._load_obj):
 
     Docstring._load_obj = staticmethod(load_obj)
 
-    # for testing omnisci-engine docs without `dbe` installation
+    # for testing hdk-engine docs without `dbe` installation
```

Comment:
Should we do something similar to below with respect to hdk?

Reply:
It should work as is because we still can fall back to `dbe` package. But it's probably worth replacing with `pyhdk` mock.